indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +70,17 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHDisplayTracks(RHConferenceBaseDisplay):
+    def _process(self):
+        page_title = get_menu_entry_by_name('program', self._conf).localized_title
+        program = track_settings.get(self.event_new, 'program')
+        return WPDisplayTracks.render_template('display.html', self._conf, event=self.event_new, page_title=page_title,
+                                               program=program)
+
+
+class RHTracksPDF(RHConferenceBaseDisplay):
+    def _process(self):
+        pdf = ProgrammeToPDF(self.event_new)
+        return send_file('programme.pdf', BytesIO(pdf.getPDFBin()), 'application/pdf')
```

Comment:
Sorry, I didn't even notice you were using a different and more British word. :P